app/themes/3column/ui/styles/theme.less

```diff
@@ -0,0 +1,533 @@
+  -webkit-border-radius: 10px;
+  -moz-border-radius: 10px;
+  -ms-border-radius: 10px;
+  -o-border-radius: 10px;
+  border-radius: 10px;
+}
+
+.btn-primary {
+  background-color: @secondary;
+}
+
+.glyphicon {
+  font-size: 1em;
+  color: @primary;
+}
+
+.container-fluid { min-height: 93%; }
+
+.navbar {
+  border-top: solid 26px @secondary;
+  height: 90px;
+  border-bottom: solid 1px #D2D2D2;
+  margin-bottom: 0;
+
+  &.navbar-default {
+    background-color: #FFF;
+  }
+
+  .navbar-brand {
+    background-image: url(/images/marklogic.png);
+    background-repeat: no-repeat;
+    background-position-y: 0px;
+    background-position-x: 0px;
+    background-size: 200px auto;
+    margin: 10px 0 0 16px;
+    width: 215px;
+  }
+
+  .navbar-form .form-control {
+    width: 325px;
+  }
+
+  .ml-search {
+    padding-right: 5px;
+    margin-top: 4px;
+
+    .form-control[type="text"] {
+      width: 525px;
+    }
+
+    .btn[type="submit"] {
+      background-color: @primary;
+      .glyphicon { color: #fff; }
+    }
+  }
+
+  .user {
+    position: absolute;
+    top: -22px;
+    right: 5px;
+    font-size: 12px;
+    color: #ff8702;
+
+    a {
+      color: #FFF
+    }
+
+    .welcome-item { padding-right: 12px; }
+
+    .navbar-form { margin-top: 34px; }
+  }
+
+
+}
+
+.panel {
+  margin-top: 5px;
+  border-radius: 0;
+
+  .panel-heading {
+    background-color: #FCFCFC;
+    border-color: #f2f2f2;
+
+    h4 {
+      margin-top: 2px;
+      margin-bottom: 2px;
+      font-size: 14px;
+      color: #444;
+      strong { font-weight: 400; }
+    }
+  }
+}
+
+.document-type-icon {
+  background-image: url('/images/filetype_sprite.png');
+  background-repeat: no-repeat;
+  display: inline-block;
+  margin: 0 5px 0 5px;
+  vertical-align: middle;
+  width: 280px;
+  height: 280px;
+  background-size: 1400px 2400px;
+
+  &.document-type-icon-md {
+    background-size: 700px 1200px;
+    width: 140px;
+    height: 140px;
+  }
+
+  &.document-type-icon-sm {
+    background-size: 350px 600px;
+    width: 70px;
+    height: 70px;
+  }
+
+  &.document-type-icon-xs {
+    background-size: 175px 300px;
+    width: 35px;
+    height: 35px;
+  }
+
+  &[document-content-type="application/msword"] { background-position: 25% 43.5%; }
+  &[document-content-type="application/vnd.ms-excel"] { background-position: 50% 43.5%; }
+  &[document-content-type="application/xml"] { background-position: 0% 14.5%; }
+  &[document-content-type="application/rtf"] { background-position: 0 85.5%; }
+  &[document-content-type="application/pdf"] { background-position: 25% 0; }
+  &[document-content-type="image/png"] { background-position: 50% 100%; }
+  &[document-content-type="image/jpeg"] { background-position: 25% 100%; }
+  &[document-content-type="image/gif"] { background-position: 75% 100%; }
+  &[document-content-type="image/tiff"] { background-position: 100% 100%; }
+  &[document-content-type="audio/mpeg"] { background-position: 25% 58%; }
+}
+
+#footer {
```

Comment:
Change this footer selector to just background-color: @white; and remove all the rest.